packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -1654,6 +1656,7 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
             <ResponsiveDocumentViewer
               isRegisterScope={this.userHasRegisterScope()}
             >
+              {console.log(declaration)}
```

Comment:
```suggestion
```

We try to avoid leaving `console.log`'s behind so other devs have a clean console to work in